ngi_pipeline/engines/piper_ngi/workflows.py

```diff
@@ -97,9 +99,9 @@ def workflow_merge_process_variantcall(*args, **kwargs):
     :rtype: str
     """
     # Same command line but with some additional options
-    return workflow_dna_variantcalling(*args, **kwargs) +  \
-            (" --variant_calling "
-             "--analyze_separately --retry_failed 2")
+    return workflow_dna_variantcalling(*args, **kwargs) + \
```

Comment:
@b97pla can you be a bit more verbose here, I think we can gain in maintainability. Maybe separate this in two lines and have an explicit if instead of the nested one into brackets. (this is personal taste)